### PR TITLE
DRAFT: Update admin dashboard docs

### PR DESCRIPTION
**Pull request by omnomagonz**
Content review of the `/administration/dashboard/` page and added some images for new features: Catalog and Installation.

### CODE REVIEW

**Comment by omnomagonz** [comment]
@pchico83 @rberrelleza FYI for your feedback as you have time

File: src/content/administration/dashboard.mdx [hunk 1]
```diff
@@ -4,75 +4,125 @@ description: Admin Dashboard
...
+We will cover these in more detail below.
+
 ## Accessing the dashboard
 
 Only accounts with the administrator role will be able to access the Administration Dashboard. The first user signing up into your Okteto instance will automatically become the initial administrator.
 
 You can promote other accounts to the administrator role in the `Users` section of the Admin Dashboard or by adding the `dev.okteto.com/super: "true"` label to the corresponding `serviceAccount` in Kubernetes.
 
-## Overview Section
+## Overview
```
Comment (pchico83) [inline src/content/administration/dashboard.mdx:37]: In the Admin UI is named "General". We should change it here or in the product, whatever makes more sense

File: src/content/administration/dashboard.mdx [hunk 2]
```diff
@@ -4,75 +4,125 @@ description: Admin Dashboard
...
-## Users Section
+With the exception of the `Settings` section, this data is informative only. The `Settings` section contains a few configurable settings to help you customize your Okteto instance as necessary.
+
+## Users
+
+From this section you can manage all users in your Okteto instance. This is a list of all users that have created an account in your Okteto instance, their respective roles, and with appropriate actions to manage their membership.
+
+:::info
+
+The `User` column is also the user's `namespace` name/id. For example, a `User` value of "darryl" means that that user's namespace will also be called "darryl".
```
Comment (pchico83) [inline src/content/administration/dashboard.mdx:61]: This is only true for the default namespace. A user might create other namespaces with random names

File: src/content/administration/dashboard.mdx [hunk 3]
```diff
@@ -95,3 +145,29 @@ ${MY_PASSWORD}
...
+
+## Catalog
+
+This section of the Admin dashboard enables you to create a [catalog](/administration/catalog/) of pre-defined development environments for anyone in your Okteto instance to deploy quickly from their `Launch Dev Environment` screen. You can reduce the amount of time it takes a developer to get up and running by defining these catalog entries so they're available for developers to select and launch with minimal to no configuration.
+
+<p align="center"><Image src="/docs/administration/images/catalog_table_view.png" alt="Catalog table view" width="900" /></p>
+
+:::info
+
+This feature is currently available only for Self-Hosted customers.
```
Comment (pchico83) [inline src/content/administration/dashboard.mdx:157]: It's available everywhere the admin view is available. That means it's only disabled in Okteto Cloud

**Review by pchico83 — APPROVED** [review]

File: src/content/administration/dashboard.mdx (hunk 1 above)
Comment (omnomagonz) [reply src/content/administration/dashboard.mdx:37]:
```suggestion
## General
```

Good catch! Thank you

File: src/content/administration/dashboard.mdx (hunk 2 above)
Comment (omnomagonz) [reply src/content/administration/dashboard.mdx:61]: @pchico83 Good catch. If a user creates 3 additional namespaces, those will only show up in the `Namespace` tab and _not_ the `Users` tab, correct?

File: src/content/administration/dashboard.mdx (hunk 3 above)
Comment (omnomagonz) [reply src/content/administration/dashboard.mdx:157]: Good catch, thank you! Will update

File: src/content/administration/dashboard.mdx (hunk 2 above)
Comment (pchico83) [reply src/content/administration/dashboard.mdx:61]: Right. The user tab only shows the user, it doesn't show information about the default namespace either

**Pull request merged**